pkg/provider/googleapps/googleapps.go

```diff
@@ -186,16 +187,16 @@ func (kc *Client) iTermCaptchaPrompt(captchaPictureURL string) (string, error) {
 	_ = b64Encoder.Close()
 
 	if os.Getenv("TERM") == "screen" {
-		fmt.Println("Detected tmux, using specific workaround...")
-		fmt.Printf("\033Ptmux;\033\033]1337;File=width=40;preserveAspectRatio=1;inline=1;:%s\a\033\\\n", buf.String())
+		log.Println("Detected tmux, using specific workaround...")
+		log.Printf("\033Ptmux;\033\033]1337;File=width=40;preserveAspectRatio=1;inline=1;:%s\a\033\\\n", buf.String())
```

Comment:
nitpicking but This could prob be a `Debug` rather than a `Printf`

Reply:
I'm actually not sure what the impact of changing the logging of the terminal emulator command would be so i've switch those back in order for this PR to be more targetted towards removing log entries. The newline behavior of `log.Printf` would likely make a mess of things.

It's possible that these would need to use `fmt.FPrintf` and explicitly specify the output stream.

Reply:
that's a great point; definitely agree on logging this being not only not useful and probably breaking things. we'd have to see the impact on leaving that one on while using credentials process